### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Accredited Representation Management: Find a Representative MVP (round 2 of 2)

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#72555]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- Lots of great stuff here! Really appreciate how you've been approaching all of this.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** In the meeting, Erin mentioned adding a heading for the search results. If you don't decide to add another heading to the page, then make sure the "Showing X of Y results..." message is coded as an H2, even if it's visually styled to look like regular text. This is consistent with the sitewide VA.gov search.
- **Should:** Some discussion in the meeting about whether/how much to auto-scroll the viewport down the page to make the search results visible after clicking the search button. For accessibility purposes, the key here is "less is more." Moving keyboard focus to the "Showing X of Y" message might be sufficient to scroll down the page all on its own. But if you want to control that behavior a little more assertively, I would recommend: 
   - Only auto-scroll enough to move the top of the results above the fold
   - Don't auto-scroll if a user has `prefers-reduced-motion` on in their browser
- **Should:** Some discussion in the meeting around the case where no results were found. I agree with Allison and Erin on having the filter dropdowns on the page in this case. I did mention maybe having an "increase search radius" button available as an option, but thinking about it more (and remembering that the "No results for..." message will come before the filter dropdowns) I don't think that's necessary here.
- **Should:** For the context you're giving about VSOs and attorneys, if it's in an additional info component, I would put it either above the "Type of representative" label, or between that text and the radio buttons. The goal is to give users the chance to encounter that context before making their choice, instead of making their choice and then having to backtrack.
   - **Consider:** If you're able to trim the copy down by a ton (like, down to one line per item), another option to consider would be the [tile variant](https://design.va.gov/storybook/?path=/docs/uswds-va-radio--default#tile) of the `va-radio-option` component. That would present the extra context about each option immediately after each option, which seems like the best possible outcome. But you definitely only want to do that if you can trim the copy down to something really lightweight.
- **Must:** The "Flag inaccuracy" link opens a modal, which means it needs to be a button. If the secondary button is too visually heavy for this situation, a few teams have been exploring tertiary button styles for some other contexts. I think the secondary style is my first choice for accessibility, but definitely reach out to Matt Dingee to talk about tertiary styles if you find that the secondary style isn't working for you.
- **Must:** In the "Flag inaccuracy" modal, I'm guessing you're required to check at least one of the boxes in order to submit, correct? If so, make sure you've got the "(* Required)" text on there.
- **Must:** In the "Flag inaccuracy" modal, the text input for "Other" needs a visible label associated with it.
- **Consider:** I forgot to mention this in the meeting, but I'd encourage you to try something a little different for after submitting an inaccuracy. Rather than a "Thank you" message in the modal, consider having the modal dismiss and then sending focus to a [slim success alert](https://design.va.gov/storybook/?path=/docs/uswds-va-alert--slim#slim) placed in context with the result that's being reported. My thoughts:
   - The alert would be persisting evidence that something happened, as opposed to the "thank you" modal just going away.
   - Placing focus on the success alert ensures that screen reader users and keyboard-only users land in a place that makes sense after reporting an inaccuracy, as opposed to landing on the "Flag inaccuracy" button again.

**Comment by oddball-lindsay** [comment]
@briandeconinck is this a note for dev?  just want to make sure i understand correctly

> **Must**: In the "Flag inaccuracy" modal, the text input for "Other" needs a visible label associated with it.

**Comment by oddball-lindsay** [comment]
>Must: The "Flag inaccuracy" link opens a modal, which means it needs to be a button. If the secondary button is too visually heavy for this situation, a few teams have been exploring tertiary button styles for some other contexts. I think the secondary style is my first choice for accessibility, but definitely reach out to Matt Dingee to talk about tertiary styles if you find that the secondary style isn't working for you.

Regarding this feedback, we have had pursued multiple discussions with the design team (including Matt Dingee) in Design Office Hours, as well as in this [Slack thread](https://dsva.slack.com/archives/C0NGDDXME/p1705681924071059) regarding a potential.  Tertiary buttons do not seem to be a viable option to explore so we are proceeding with the secondary button as originally recommended.

Mobile:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/142453186/bab690de-3e32-4cf8-8580-953e5121f997)

Desktop:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/142453186/f83072b9-b5ad-4fb4-8d47-7d718c14486d)

@briandeconinck @allison0034 our next interaction will be in Staging Review.   We'll be picking up development shortly based on these designs and want to double check - do you anticipate any launch-blocking issues with this path forward?

**Comment by allison0034** [comment]
I do not from a design perspective, Matt told you to implement this UI at design office hours. That being said, make sure you use the components that are available in Storybook.

**Comment by briandeconinck** [comment]
@oddball-lindsay Sorry for the slow reply! No anticipated launch blockers. 🙂
